inc/Engine/Optimization/RUCSS/Frontend/Subscriber.php

```diff
@@ -123,6 +135,35 @@ public function treeshake( string $html ) : string {
 		return $html;
 	}
 
+	/**
+	 * Schedules RUCSS to retry pages with missing CSS files.
+	 * Retries happen after 30 minutes.
+	 *
+	 * @return void
+	 */
+	public function schedule_rucss_retry() {
+		$scheduled = wp_next_scheduled( self::CRON_NAME );
+
+		if ( $scheduled ) {
+			return;
+		}
+
+		wp_schedule_single_event( time() + ( 60 * 30 ), self::CRON_NAME );
```

Comment:
I have two comments here:-
1. What do u think of using this constant to make it more readable
```suggestion
		wp_schedule_single_event( time() + ( 0.5 * HOUR_IN_SECONDS ), self::CRON_NAME );
```

2. I think we need to add filter here so customers/helper plugins can change this period.

What do u think ?

Reply:
Yeah. I will apply the first change.

However, for the filter I think we shall wait and implement it only if needed. If I remember well I had this talk with @Tabrisrp and he said to do it like this for now.